core/market/decentralized/src/matcher/matcher.rs

```diff
@@ -264,8 +274,8 @@ async fn on_offer_received(db: DbExecutor, msg: OfferReceived) -> Result<Propaga
         Result::<_, MatcherError>::Ok(propagate)
     }
     .await
-    .or_else(|e| {
-        let reason = StopPropagateReason::Error(format!("{}", e));
+    .or_else(|error| {
```

Comment:
why `e` is not enough? Lets be consistent across our module, 

All below stats are with three usages from here

```
~/git/yagna market-decentralized/query-events (04ecd2c2)$ grep --include '*.rs' -rni '|e|' agent/ core/ exe-unit/ service-bus/ test-utils/ utils/ | wc
      75     336    8157
~/git/yagna market-decentralized/query-events (04ecd2c2)$ grep --include '*.rs' -rni '|error|' agent/ core/ exe-unit/ service-bus/ test-utils/ utils/ | wc
      32     132    3282
```

and

```
~/git/yagna market-decentralized/query-events (04ecd2c2)$ grep --include '*.rs' -rni '|error|' core/market/decentralized/ | wc
       5      18     505
~/git/yagna market-decentralized/query-events (04ecd2c2)$ grep --include '*.rs' -rni '|e|' core/market/decentralized/ | wc
      10      44    1209
```